src/refract/JSONSchemaVisitor.cc

```diff
@@ -222,23 +222,39 @@ namespace refract
         }
     }
 
+    ObjectElement* JSONSchemaVisitor::definitionFromVariableProperty(JSONSchemaVisitor& renderer)
+    {
+        ObjectElement* definition = new ObjectElement();
+
+        definition->push_back(new MemberElement("type", new StringElement("object", IElement::rCompact), IElement::rCompact));
```

Comment:
And I do ... so this brings again the bloody style issue, what is our column limit? Is it 80, 120, 280? According to the clang format it is 120, https://github.com/apiaryio/snowcrash/pull/380/files#diff-69a68f7ce1ea3687a38d8e4705e6fa48R25